openmdao/docs/developer_docs/repo_guide/advanced_operations.rst

```diff
@@ -0,0 +1,234 @@
+            storing result as deploy.rsa.enc
+            storing secure env variables for decryption
+            ** GET "settings/env_vars/?repository_id=XXXXXXX"
+            **   took 0.051 seconds
+            ** POST "settings/env_vars/?repository_id=XXXXXXX" "{\"env_var\":{\"public\":false,\"name\":\"encrypted_67eXXXXXXXXX_key\",\"value\":\"?????????????????????????????\"}}"
+            **   took 0.064 seconds
+            ** GET "settings/env_vars/?repository_id=XXXXXXX"
+            **   took 0.049 seconds
+            ** POST "settings/env_vars/?repository_id=XXXXXXX" "{\"env_var\":{\"public\":false,\"name\":\"encrypted_67eXXXXXXXXX_iv\",\"value\":\"??????????????????????????????\"}}"
+            **   took 0.057 seconds
+
+            Make sure to add deploy.rsa.enc to the git repository.
+            Make sure not to add deploy.rsa to the git repository.
+            Commit all changes to your .travis.yml.
+            ** Deleting "/Users/xxxxxxxx/.travis/error.log"
+            ** Storing "/Users/xxxxxxxx/.travis/config.yml"
+
+
+       The command is attempting to POST those env vars, but they don’t seem to make it to the OpenMDAO account.
+       However, the name and value are right there in the debug output, so they can easily be copied and pasted into the Travis CI web
+       interface (https://travis-ci.org/<user>/<project>/settings ). Creating these env variables must be done, because the
+       openssl decrypt command is going to refer to those env vars in the `-K` and `-iv` arguments.
+
+
+**3. Copy Key to Web Server**
+    To copy the key over to your web server.  In the specific case of OpenMDAO, let’s take a moment to explore what needs to be done on Webfaction.
+
+    A. Need to create a web server application on Webfaction (for local NASA users).
+
+     1. Go to panel.webfaction.com,
+     2. Click Domains/Websites,
+     3. Choose the Applications tab.
+     4. Click the Add New Application button.
+     5. Give your new app an appropriate name, for our example, I chose “openmdaodocs.”
+     6. Make the app as type “Static Only (no .htaccess)."
+     7. Click on Websites, choose openmdao_org,
+     8. Choose, “reuse an existing application” and then pick your newapp and give it a url.
+     9. After a moment, a folder will appear on webXXX, under ~/webapps/<name>, that is accessible at openmdao.org/<url>. Keep in mind that webXXX.webfaction.com:webapps/<name> will be your path to copy your docs to.
+
+    B. Need to copy the public key generated above to our Webfaction server to allow passwordless entrance.
+
+     1. On webXXX, in the ~/.ssh folder, there is a file called authorized_keys.
+     2. Copy the contents of the travis_deploy_rsa.pub as an entry into the authorized_keys file.
+
+**4. Modify the YAML Further**
+
+   A. Late in the `before_install` section add this line:
+        `- echo -e "Host <server address>\n\tStrictHostKeyChecking no\n" >> ~/.ssh/config`
```

Comment:
looks like you need to escape these backslashes for it to display correctly in the built page